lms/models/key.py

```diff
@@ -0,0 +1,45 @@
+from uuid import uuid4
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import UUID
+
+from lms.db import BASE
+from lms.models import CreatedUpdatedMixin
+
+
+class Key(CreatedUpdatedMixin, BASE):
```

Comment:
Storing public/private keys here.

They are used to sign messages sent to the LMS for example for deeplinking and grading.